examples/avmnist_multimodal/config.py

```diff
@@ -0,0 +1,68 @@
+"""
+Default configurations for avmnist dataset using Multimodal Deep Learning (MMDL).
+"""
+
+from yacs.config import CfgNode as CN
+
+# -----------------------------------------------------------------------------
+# Config definition
+# -----------------------------------------------------------------------------
+
+_C = CN()
+
+# -----------------------------------------------------------------------------
+# Dataset
+# -----------------------------------------------------------------------------
+_C.DATASET = CN()
+_C.DATASET.ROOT = "avmnist"
+_C.DATASET.NAME = "data.zip"
```

Comment:
That is an uninformative name. What data is not data? Can you refer to other examples?